web/crux/src/app/image/image.dto.ts

```diff
@@ -26,6 +55,9 @@ export class ImageDto {
 
   @Type(() => BasicRegistryDto)
   registry: BasicRegistryDto
+
+  @ValidateNested()
+  validation: ImageValidation
```

Comment:
COAL's ruleset should not bleed into crux's API structure. At this point it should only exist as labels IMHO.